### DISCUSSION TITLE
Form checkbox value is not changed on the first click

### DISCUSSION THREAD

**Discussion by borachoidev** [mantinedev/mantine#4489] [Q&A]
<img width="478" alt="스크린샷 2023-06-30 오후 2 11 52" src="https://github.com/mantinedev/mantine/assets/66766189/9417474e-a54d-46f9-946d-2b935ac1b2d8">

When I click on the checkbox component, the checkbox value does not change. However, after the second click, it works well. Only the first click does not work. I have already added `{ type: 'checkbox' }`, but it still does not work.

```jsx
   <Checkbox
        label="agree" 
        {...form.getInputProps('agree', { type: 'checkbox' })}
      />
```

How can I fix this issue?

**Comment by rtivital** [comment]
It works correctly on codesandbox – https://codesandbox.io/s/compassionate-sid-742p4z?file=/src/App.tsx
